apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -137,6 +137,8 @@ const SwapForm = () => {
     onNoRates,
     ...(locationState as object),
     providers: storedProviders || undefined,
+    timeout: 10000,
```

Comment:
As we are going to use this const for more calls....perhaps we could save it in a constant

Reply:
Good idea! 😄